libs/utils/utils.js

```diff
@@ -380,35 +380,34 @@ export async function loadBlock(block) {
 
 export function decorateSVG(a) {
   const { textContent, href } = a;
-  const altTextFlagIndex = textContent.indexOf('|');
-  const sanitizedTextContent = altTextFlagIndex === -1
-    ? textContent
-    : textContent?.slice(0, altTextFlagIndex).trim();
-  const ext = sanitizedTextContent?.substring(sanitizedTextContent.lastIndexOf('.') + 1);
-  if (ext !== 'svg') return;
-
-  const altText = altTextFlagIndex === -1
-    ? ''
-    : textContent.substring(textContent.indexOf('|') + 1).trim();
-  const img = document.createElement('img');
-  img.setAttribute('loading', 'lazy');
-  img.src = localizeLink(sanitizedTextContent);
-  img.alt = altText;
-  const pic = document.createElement('picture');
-  pic.append(img);
-
+  if (!(textContent.includes('.svg') || href.includes('.svg'))) return a;
```

Comment:
nit: you could use `href.endsWith('.svg')` for href checking.

Reply:
...unless someone does: 
`https://.../my-icon.svg#_blank`
or
`https://.../my-icon.svg#_dnt`